tests/v2/validation/prime/README.md

```diff
@@ -0,0 +1,19 @@
+# Prime Configs
+
+## Getting Started
+Your GO suite should be set to `-run ^TestPrimeVersionTestSuite$`. You can find any additional suite name(s) by checking the test file you plan to run.
+
+In your config file, set the following:
+```json
+"rancher": { 
+  "host": "rancher_server_address",
+  "adminToken": "rancher_admin_token"
+  ...,
+}
+"prime": {
+  "rancherVersion": "<version_or_commit_of_rancher",
```

Comment:
nit..missing a `>` bracket at the end of the `rancherVersion`. Also, if you enter the version, you have to put a leading `v`, correct?

Reply:
correct, however you can also enter a commit version so I didn't want to confuse the user if they plan to enter the commit version (which doesn't have a `v` in front).